excercises/02. Function Exercises/01. Calculating Average/Calculate_average_Morme_02.py

```diff
@@ -0,0 +1,16 @@
+def getNumber1():
+    return int(input("Podaj pierwszą liczbę: "))
+
+def getNumber2():
+    return int(input("Podaj drugą liczbę: "))
+
+def getNumber3():
+    return int(input("Podaj trzecią liczbę: "))
+
+def calculateMiddle():
+    middle = (getNumber1()+getNumber2()+getNumber3())/3
+    return print(float(round(middle, 2)))
```

Comment:
- middle jest floatem
- potem zaokrąglasz wartośc middle do 2 miejsc po przecinku (i nadal jest floatem, bo floaty mają coś po przecinku)
- potem rzutujesz to do floata - nie jest to potrzebne